qldbdriver/result_test.go

```diff
@@ -102,31 +101,28 @@ func TestResult(t *testing.T) {
 				result.index = 0
 				result.pageToken = &mockToken
 				mockService := new(mockResultService)
-				mockService.On("fetchPage", mock.Anything, mock.Anything, mock.Anything).Return(&mockFetchPageResultWithStats, nil)
+				mockService.On("fetchPage", mock.Anything, mock.Anything, mock.Anything).Return(&fetchPageResultWithStats, nil)
 				result.communicator = mockService
 
-				var mockTimingInformation *TimingInformation = nil
-				var mockConsumedIOs *IOUsage = nil
-
 				// Default page
 				assert.True(t, result.Next(&transactionExecutor{nil, nil}))
-				assert.Equal(t, mockTimingInformation, result.timingInformation)
-				assert.Equal(t, mockConsumedIOs, result.consumedIOs)
+				assert.Equal(t, new(int64), result.statementStatistics.timingInformation.GetProcessingTimeMilliseconds())
```

Comment:
Same here: just `0` is easier to scan

Reply:
The Go AWS SDK is returning [IOUsage/TimingInformation structs](https://github.com/aws/aws-sdk-go/blob/v1.37.15/service/qldbsession/api.go#L659-L667) with fields of type: int64. The driver passes these values through.

If I set it to `0`, the test fails with:
```
Expected :int(0)
Actual   :int64(0)
```